docs/content/getting-started/_index.md

```diff
@@ -92,17 +125,155 @@ class App extends Component {
 
 ## 4. Add attributes 
 
-You may choose to add attributes such as `priority` or `color` as needed to adjust usage of palette color, context, or priority.
+You may choose to add attributes such as `pfe-variant`, `pfe-priority` or `pfe-color` as needed to adjust usage of general theme, palette color, or priority. Check the readme file of a component to see what attributes are supported. Or check out the [Storybook](https://patternfly.github.io/patternfly-elements/demo) to preview how the attributes work.
 
-```html
-  <pfe-card color="darkest">
-    <h2 slot="header">Dark card</h2>
-    <p>Lorem ipsum.</p>
-    <pfe-cta color="complement" priority="primary" on="dark" slot="footer">
-        <a href="#">Important call-to-action</a>
-    </pfe-cta>
-  </pfe-card>
+#### General
+
+  Priority & variant attributes change style styles globally:
+
+  ```
+  <pfe-cta pfe-priority="primary">...</pfe-cta>
+  <pfe-tabs pfe-variant="earth">...</pfe-tabs>
+  ```
+
+  **Full list:**
+
+  *   `pfe-priority`
+  *   `pfe-variant`
+
+
+#### Specific
+
+Color or align attributes change specific properties
+
+```
+<pfe-cta pfe-color="accent">...</pfe-cta>
 ```
 
 
+#### **Full list:**
+
+*   `pfe-align`
+*   `pfe-color`
+*   `vertical`
+*   `horizontal`
+
+
+## 5. Use CSS variables as needed
```

Comment:
Suggest: Use CSS variables to provide hooks for customizing the component or as a means of updating styles based on attributes

It's a long title I know, maybe there's a more succinct way to communicate those two things.